tools/topology/topology1/development/sof-imx8-src-wm8960.m4

```diff
@@ -27,6 +27,8 @@ include(`platform/imx/imx8.m4')
 # PCM0 <----> SRC <----> volume <-----> `SAI_INDEX' (`CODEC')
 #
 
+ifdef(`DDMA_DOMAIN', `define(`SCHEDULE_DOMAIN', SCHEDULE_TIME_DOMAIN_DMA)', `define(`SCHEDULE_DOMAIN', SCHEDULE_TIME_DOMAIN_TIMER)')
+
```

Comment:
Is this a typo? `DDMA_DOMAIN` -> `DMA_DOMAIN`

Reply:
> Is this a typo? `DDMA_DOMAIN` -> `DMA_DOMAIN`
> 
It's a typo. I've just fixed it, Thanks!

> Also, it looks like we can move this definition inside include(`platform/imx/imx8.m4') so we can only write it once.
>
I prefer to keep it here since in imx8.m4 are more generic stuff.
Also, this will soon be removed, since 8MP will be switched to TIMER domain as 8QM, 8QXP and 8ULP.